### PR TITLE
Use click to implement CLI for the tool 

### PR DESCRIPTION
**Pull request by signalpillar**
With all the advantages of click, the code doesn't differ much from what we had
before.  Just is a bit more declarative.

### CODE REVIEW

File: .circleci/config.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+version: 2
```
Comment (signalpillar) [inline .circleci/config.yaml:1]: Can't test it on forked repo at the moment.

**Comment by Deepwalker** [comment]
I'm not sure about `click`. Yes, there is nothing viable in stdlib for this, this is why pundle have its own 15 lines custom thing.
But we avoid any dependencies, because we install pundle to site-packages, and it must not install anything else.
Only way is to vendor lib to pundle package like pip do.

**Comment by signalpillar** [comment]
@Deepwalker sorry for my late reply. I understand now why you had own implementation. It makes sense. We can vender if you are ok with that. 
Another way is to improve your custom implementation by using `argparse`. I'll check that option too.

**Comment by signalpillar** [comment]
Closing in favour of #27

**Pull request closed without merging**